frontend/src/components/VendorIntroduction.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import '../styles/vendor-introduction.css';
+import { v4 } from 'uuid';
+
 
 const VendorIntroduction = ({ introductionLong }) => {
   return (
     <>
       {introductionLong && (
         <div className='introduction'>
           <h2>Bemutatkozás</h2>
-          <p>{introductionLong}</p>
+          {introductionLong.split('\n').map((introduction) => (
+            <p key={v4()}>{introduction}</p>
```

Comment:
ez is egy megközelítés, viszont futás időben költséges. nem lenne egyszerűbb CSS-el megoldani?
`white-space: pre-wrap;`
tudomásom szerint ez feloldja a sortöréseket :)